swagger_sdk_gen.sh

```diff
@@ -3,8 +3,8 @@ if $Build_Codegen ; then
 fi
 if [ "$Branch" = "snapshot" ]
   then
-    url="http://newapi-nightly.capillary.in/swagger.json"
-    version="http://newapi-nightly.capillary.in/version.json"
+    url="http://crm-nightly-new.cc.capillarytech.com/swagger.json"
```

Comment:
change it to https, http wont work in cc. also the url is wrong. sub domain needs to be present. 
correct url is - https://intouch-api-swagger.crm-nightly-new.cc.capillarytech.com/swagger.json